src/styles/index.scss

```diff
@@ -1,3 +1,226 @@
+.header__nav-link:hover {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  background-color: #00acdc;
+  border-radius: 8px;
+  height: 4px;
+  bottom: 0;
+  right: 0;
+  left: 0;
+}
+
+.catalog {
+  width: 100%;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  gap: 46px 48px;
+  justify-content: center;
+  align-items: center;
+  padding: 50px 40px;
+}
+
+@media screen and (min-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .catalog {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media screen and (min-width: 1024px) {
+  .catalog {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.card {
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+  transition: transform 0.3s ease-in-out;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  display: flex;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  margin: 0 0 4px;
+  line-height: 18px;
+  font-size: 12px;
+  font-weight: 500;
+  color: #060b35;
+}
+
+.card__title:hover {
+  color: #34568b;
+}
+
+.card__code {
+  line-height: 14px;
+  margin: 0 0 24px;
+  text-align: left;
+  width: 100%;
+  color: #616070;
+  font-size: 10px;
+  font-weight: 400;
+}
+
+.card__reviews {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0;
+  padding-bottom: 16px;
+  width: 166px;
+  height: 16px;
+  line-height: 14px;
+}
+
+.card__reviews-p {
+  font-size: 10px;
+  color: #060b35;
+  margin: 0;
+}
+
+.card__price {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.card__price-title {
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__button {
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 40px;
+  border: 1px solid #00acdc;
+  background-color: #00acdc;
+  border-radius: 5px;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #fff;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+}
+
+.card__button:hover {
+  background-color: #fff;
+  color: #00acdc;
+  border: 1px solid #00acdc;
+}
+
+.card:hover {
+  transform: scale(1.2);
+}
+
+.card__stars {
+  display: flex;
+  width: 96px;
+  flex-direction: row;
+  justify-content: space-around;
+}
+
+.card__stars-star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+  background-size: 80%;
+  width: 16px;
+  height: 16px;
+}
+
+.card__stars-star:nth-child(-n + 1),
+.card__stars-star:nth-child(-n + 2),
+.card__stars-star:nth-child(-n + 3),
+.card__stars-star:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
 }
```

Comment:
Checklist item: 'Avoid using nth-child selector for BEM-elements'. It's better to add a modifier class to the '.card__stars-star' elements that should be active (e.g., '.card__stars-star--active') instead of using the 'nth-child' selector. This makes it easier to manage the active state dynamically with JavaScript.